src/include/query_stats.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <cstdint>
+
+namespace duckdb {
+
+struct ReqStats {
+	float elapsed_sec;
+	uint64_t read_bytes;
```

Comment:
Do we perhaps want to remove `read_bytes` and `read_rows` until they are used? Otherwise people will just be confused

Reply:
FYI `read_bytes` and `read_rows` are used by the UI to display the results and emulate Clickhouse query stats

Reply:
But aren't hey always 0?

Reply:
They shouldn't be when using the JSONCompact format but I'll re-check